packages/atomic/cypress/integration/facets-v1/color-facet/color-facet-actions.ts

```diff
@@ -0,0 +1,19 @@
+import {TestFixture, addTag, TagProps} from '../../../fixtures/test-fixture';
+import {ColorFacetSelectors} from './color-facet-selectors';
+
+export const defaultNumberOfValues = 8;
+export const label = 'File Type';
+export const field = 'filetype';
+
+export const customSript = `atomic-color-facet::part(value-YouTubeVideo) {
```

Comment:
I would prefer if all exported variables had slightly more specific names. Otherwise, they will get suggested by the code editor while writing unrelated tests.